Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
> Idea is to serve close to the same v8 configuration that served on debian/ubuntu via libv8

Don't Debian/Ubuntu ship the whole `icu` as part of the system natively though? Am I misremembering there?

> May it lead to some problems under homebrew infrastructure?

No. It just means the bottle can't be poured for every prefix, only the one it was built for. This isn't uncommon sadly.